src/components/panels/WebcamPanel.vue

```diff
@@ -63,6 +63,9 @@
                     <template v-else-if="currentCam.service === 'webrtc-camerastreamer'">
                         <webcam-webrtc-camerastreamer :cam-settings="currentCam" />
                     </template>
+                    <template v-else-if="currentCam.service === 'webrtc-mediamtx'">
+                        <webcam-webrtc-mediamtx :cam-settings="currentCam" :collapsible="$route.fullPath !== '/cam'" />
```

Comment:
```suggestion
                        <webcam-webrtc-mediamtx :cam-settings="currentCam" />
```

this variable is not used in the component anymore